### PR TITLE
updated web based gmap api scanner

### PR DESCRIPTION
**Pull request by Anugrahsr**

### CODE REVIEW

File: API Key Leaks/README.md [hunk 1]
```diff
@@ -134,4 +136,4 @@ $ AspDotNetWrapper.exe --decryptDataFilePath C:\DecryptedText.txt
 * [Finding Hidden API Keys & How to use them - Sumit Jain - August 24, 2019](https://medium.com/@sumitcfe/finding-hidden-api-keys-how-to-use-them-11b1e5d0f01d)
 * [Private API key leakage due to lack of access control - yox - August 8, 2018](https://hackerone.com/reports/376060)
 * [Project Blacklist3r - November 23, 2018 - @notsosecure](https://www.notsosecure.com/project-blacklist3r/)
-* [Saying Goodbye to my Favorite 5 Minute P1 - Allyson O'Malley - January 6, 2020](https://www.allysonomalley.com/2020/01/06/saying-goodbye-to-my-favorite-5-minute-p1/)
\ No newline at end of file
+* [Saying Goodbye to my Favorite 5 Minute P1 - Allyson O'Malley - January 6, 2020](https://www.allysonomalley.com/2020/01/06/saying-goodbye-to-my-favorite-5-minute-p1/)
```
Comment (PinkDev1) [inline API Key Leaks/README.md:139]: nothing changed????

**Review by Ogungodspower5 — APPROVED** [review]
3GkHwKSDrsLcFpDCbMAXKWpGZ4u9DNsYXn

**Comment by Ogungodspower5** [comment]
3GkHwKSDrsLcFpDCbMAXKWpGZ4u9DNsYXn

File: API Key Leaks/README.md (hunk 1 above)
Comment (Anugrahsr) [reply API Key Leaks/README.md:139]: added line 36
web-based gmap api scanner

**Pull request closed without merging**